timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -181,4 +193,24 @@ services:
         http: POST /sw/{namespace}
         args:
           namespace: string
-          request: LockWatchRequest
\ No newline at end of file
+          request: LockWatchRequest
+  CrossClientBatchedConjureTimelockService:
+    name: Cross Client Batched Timelock Service
+    default-auth: header
+    package: com.palantir.atlasdb.timelock.batch.api
```

Comment:
as above: don't think batching is correct. multiclient maybe - or can this just go in the same package?